src/common/offline_transformations/src/pruning/propagate_masks.cpp

```diff
@@ -1432,6 +1432,42 @@ class ngraph::pass::mask_propagation::Transpose : public MatcherPass {
     }
 };
 
+static ngraph::Mask::Ptr create_split_mask(ngraph::Mask::Ptr input_mask,
+                                           int64_t axis,
+                                           uint64_t split_start,
+                                           uint64_t split_end) {
```

Comment:
Please consider adding a comment like here
https://github.com/openvinotoolkit/openvino/blob/4d11c37636eab540f33923ce7f8aa0aee15566e9/src/common/offline_transformations/src/pruning/propagate_masks.cpp#L1091-L1094.

I would recommend to reflect meaning of this function in its name: it isn't just creating split mask but it's adding callbacks to input/output masks besides. I believe that name like `create_connect_split_output_mask` would gain code simplicity. @mateusztabaka, what do you think?

Reply:
done